hyperhtml/documentation/index.html

```diff
@@ -91,22 +91,21 @@
           </div>
           <div class="column">
             <section id="introduction">
-              <h2>Introduction</h2><hr>
-              <h3 id="introduction-0">What is hyperHTML ?</h3>
+              <h2>Introduction</h2>
+              <h3 id="introduction-0">What is hyperHTML?</h3>
+              <hr>
               <p>
-                100% based on Web standards, hyperHTML is
-                a zero-dependencies, fully cross platform library,
-                suitable to create <strong>declarative</strong> and
+                hyperHTML is a Web standard compliant, zero-dependency, fully cross platform library
+                suitable for <strong>declarative</strong> and
                 <strong>reactive</strong> Web Applications.
               </p>
               <p>
-                There is basically nothing new to learn with hyperHTML,
-                or better, nothing more than standard JavaScript, HTML, or CSS.
+                There is nothing new to learn with hyperHTML other than standard JavaScript, HTML, or CSS.
               </p>
               <p>
                 Its core features are built on top of <a href="https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals">template literals</a>,
-                where every interpolation is addressed once, as unique DOM operation, and updated at light-speed every time it's needed.
-                You can check <a href="https://webreflection.github.io/hyperHTML/test/tick/">the most basic hyperHTML example</a> out,
+                where every interpolation is addressed once, as a unique DOM operation, and updated at lightning-speed when needed.
```

Comment:
I guess what you're trying to illustrate is that, this thing is fast on re-renders, maybe a better way to describe that...?